habitat-baselines/habitat_baselines/rl/hrl/skills/oracle_pick.py

```diff
@@ -23,6 +23,7 @@ class OraclePickPolicy(NnSkillPolicy):
 
     GRAB_ID = 1
     RELEASE_ID = 0
+    PICK_ID = 1
```

Comment:
Why would you have a repeated GRAB_ID and PICK_ID with same value? BTW, why would you want to rename "grab" to "pick"? Can you explain this?

Reply:
Sorry for the confusion. Actually, I have modify the class that pick policy only uses PICK_ID. GRAB_ID and RELEASE_ID is useless here. I will push another commit to fix it.